x/inflation/v1/keeper/integration_test.go

```diff
@@ -96,9 +96,9 @@ var _ = Describe("Inflation", Ordered, func() {
 				params := s.app.InflationKeeper.GetParams(s.ctx)
 				params.EnableInflation = true
 				params.InflationDistribution = types.InflationDistribution{
-					UsageIncentives: sdk.NewDecWithPrec(533333334, 9), // 0.53 = 40% / (1 - 25%)
-					StakingRewards:  sdk.NewDecWithPrec(333333333, 9), // 0.33 = 25% / (1 - 25%)
-					CommunityPool:   sdk.NewDecWithPrec(133333333, 9), // 0.13 = 10% / (1 - 25%)
+					StakingRewards:  sdk.NewDecWithPrec(333333333, 9),
+					CommunityPool:   sdk.NewDecWithPrec(666666667, 9),
+					UsageIncentives: math.LegacyZeroDec(), // Deprecated
 				}
 				_ = s.app.InflationKeeper.SetParams(s.ctx, params)
```

Comment:
How are we going to handle this param? Is this going to be phased out in a follow up release?

Reply:
yes, for the new token economics inflation module